exporter/exporter_test.go

```diff
@@ -43,3 +43,33 @@ func TestAggregatedMetricValues(t *testing.T) {
 		t.Errorf("expected after aggregation: %#v, got: %#v", expected, aggregated)
 	}
 }
+
+func TestAggregatedMetricValuesEmptyLabels(t *testing.T) {
+	// labels can be empty if we skipped a label, for example with regexp
```

Comment:
Labels can definitely be empty (zero labels). The values cannot be (zero length string label).